<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Asetukset</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="footerButtonLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Käytä</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="defaultSettingsButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Oletusasetukset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Peruuta</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="proxySettingsMainVLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Määritä mahdolliset Proxy-palvelimen osoitteet</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="HTTPAddressLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HTTP-osoite:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="HTTPAddressLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhLowercaseOnly|Qt::ImhNoAutoUppercase|Qt::ImhUrlCharactersOnly</set>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>47</number>
        </property>
        <property name="placeholderText">
         <string notr="true">http://proxy.example.org:8080</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">http</string>
        </property>
        <property name="settingsGroup" stdset="0">
         <string notr="true">proxySettings</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="HTTPSAddressLabel">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>HTTPS-osoite:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="HTTPSAddressLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhLowercaseOnly|Qt::ImhNoAutoUppercase|Qt::ImhUrlCharactersOnly</set>
        </property>
        <property name="inputMask">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="maxLength">
         <number>47</number>
        </property>
        <property name="placeholderText">
         <string notr="true">https://proxy.example.org:8080</string>
        </property>
        <property name="settingsKey" stdset="0">
         <string notr="true">https</string>
        </property>
        <property name="settingsGroup" stdset="0">
         <string notr="true">proxySettings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
